nextflow/main.nf

```diff
@@ -0,0 +1,32 @@
+params.read_triplets = "$projectDir/test_data/reads/reads_{1,2,3}*.gz"
+params.outdir = "results"
+
+process UMI_INJECTOR {
+    container 'umi-injector'
```

Comment:
This remains to be fixed

Reply:
Are you sure that this "tool" should be published on Bioconda? After all, it is just a hacky, temporary solution...but admittedly I don't know if I will manage to implement something better in Rust.

A hard limit with umi-injector is, that it will never be able to flexibly accommodate different read naming schemes, since a regex can't be passed into awk's `match` function as a variable. So one would need to change the script to make it work with read naming patterns that do not comply with the Illumina standard.  

![code_lifespan_2x](https://user-images.githubusercontent.com/6963520/228516580-f30d9f8a-d5cf-47da-8dcf-3a253df2d116.png)